@AH-64D Apache Longbow/Addons/fza_ah64_audioSystem/functions/fn_addWarning.sqf

```diff
@@ -0,0 +1,29 @@
+/* ----------------------------------------------------------------------------
+Function: fza_audioSystem_fnc_addWarning
+
+Description:
+
+Parameters:
+    _heli: object - the apache the player is in
+
+Returns:
+    No returns
+
+Examples:
+
+Author:
+    Rosd6(Dryden)
+---------------------------------------------------------------------------- */
+params ["_heli", "_priority", "_Reference"];
+
+private _wrnMsg = _heli getVariable "fza_audio_warning_message";
+
+if (_wrnMsg == "") then {
+    _heli setvariable ["fza_audio_ase_message", _this];
+};
+
+private _FuncHook = _heli getVariable ["fza_audio_FuncHook", scriptNull];
```

Comment:
Please change _FunfHook to _funcHook for both the function name and variable name in lines 26, 27 and 28